metis-dereference/metis-dereference-common/src/main/java/eu/europeana/metis/dereference/Vocabulary.java

```diff
@@ -62,15 +83,26 @@
   @Indexed(options = @IndexOptions(unique = true))
   private String name;
 
-  private ContextualClass type;
-
   @XmlElement
-  public String getUri() {
-    return uri;
+  public Set<String> getUris() {
+    return Collections.unmodifiableSet(this.uris);
   }
 
-  public void setUri(String uri) {
-    this.uri = uri;
+  public void setUris(Collection<String> uris) {
+    this.uris = new HashSet<>(uris);
+
+    // Also take care of backwards compatibility
+    final String sampleUrl = uris.iterator().next();
```

Comment:
Isn't it better to use the this.uris instead of the method parameter?

Reply:
I don't think it matters in this case... but no problem.